src/game_board/game_board.html

```diff
@@ -12,9 +12,9 @@
                     ng-class="{'selected': info.selected, 'targetable': doingAction && targets.indexOf(info.id) !== -1}">{{info.name}}</h4>
                 <ul style="list-style: none outside none; margin: 0; padding: 0;">
                     <li ng-repeat="(name, value) in info.properties">
-                        <b>{{name}}</b>: {{value}}
+                        <b>{{name | capitalize}}</b>: {{value}}
```

Comment:
I first tried to do the capitalization with CSS, but I couldn't get the `::first-letter` thing to work. `text-transform: capitalize` doesn't work since it Capitalizes Every Word.